src/ToDo/Presentation/Dialogs/RenameListViewModel.cs

```diff
@@ -2,5 +2,21 @@ namespace ToDo.Presentation;
 
 public partial class RenameListViewModel
 {
-    
-}
\ No newline at end of file
+	private INavigator Navigator { get; }
```

Comment:
There is no need for the RenameCommand anymore normally as it's done by the extension with the navigation normally in [RenameListFlyout.xaml](https://github.com/unoplatform/uno.todo/pull/150/files#diff-655db2e39acf518fbaa3b870cde099dd94a39d131ff35fba91e1195afaab1d3f)

```
uen:Navigation.Request="-"
uen:Navigation.Data="{Binding Text, ElementName=ListNameText}"
````